app/components/Article/KeepGoing/keepGoing.module.css

```
@@ -4,3 +4,14 @@
   padding: var(--spacing-32);
   margin-bottom: var(--spacing-56);
 }
+
+@media (max-width: 640px) {
+  .flex_dynamic {
```

Other two word class names seem to be separated with a hyphen; probably best to stay consistent here unless there is a special reason.
```suggestion
  .flex-dynamic {
```

---

@jrhender  In react it's imported as styles.module, hyphen is not a valid character in a JS variable

---

Ok, thanks for the explanation!